packages/@coorpacademy-components/src/molecule/learner-skill-card/index.js

```diff
@@ -0,0 +1,180 @@
+import React, {useCallback, useState} from 'react';
+import {convert} from 'css-color-function';
+import {get, getOr} from 'lodash/fp';
+import PropTypes from 'prop-types';
+import Icon from '../../atom/icon';
+import ButtonLink from '../../atom/button-link';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const LearnerSkillCard = (props, context) => {
+  const {
+    'aria-label': ariaLabel,
+    skillTitle,
+    skillAriaLabel,
+    metrics,
+    onReviewClick,
+    onExploreClick
+  } = props;
+  const {skillCourses, skillQuestions, completedCourses = 0} = metrics;
+  const {skin, translate} = context;
+  const [hovered, setHovered] = useState(false);
+  const primarySkinColor = getOr('#0061FF', 'common.primary', skin);
+
+  const review = translate('Review');
+  const explore = translate('Explore');
+  const courses = translate('courses');
+  const questions = translate('questions');
+  const skillFocus = translate('skill_focus');
+  const coursesCompleted = translate('courses_completed');
```

Comment:
(I might rename with Locale suffix to disambiguitate in the component building around line 100 (ex [line 104](https://github.com/CoorpAcademy/components/pull/2846/files#diff-7a6de17128fffd61ef831601136afb36293e13abca97f66b1342f2a7972d047aR104)